lib/qualys/element.rb

```diff
@@ -99,15 +99,20 @@ def cleanup_html(source)
       result.gsub!(/<p>/i, "\n\n")
       result.gsub!(/<br>/i, "\n")
       result.gsub!(/          /, "")
-      result.gsub!(/<a href=\"(.*?)\" target=\"_blank\">(.*?)<\/a>/i) { "\"#{$2.strip}\":#{$1.strip}" }
+      result.gsub!(/<a href=\"(.*?)\"\s?target=\"_blank\">(.*?)<\/a>/i) { "\"#{$2.strip}\":#{$1.strip}" }
       result.gsub!(/<pre>(.*?)<\/pre>/im) { |m| "\n\nbc.. #{$1.strip}\n\np.  \n" }
       result.gsub!(/<b>(.*?)<\/b>/i) { "*#{$1.strip}*" }
+      result.gsub!(/<b>|<\/b>/i, "")
```

Comment:
why is this needed and wouldn't it be caught by the the previous line?

Reply:
In the sample files we got from teams, there are several instances of `<b>` tags that are orphaned (e.g. they don't match up open/closing). The previous line finds cases where there is an opening and a closing tag to apply the bolding formatting, then this line cleans up any remaining `<b>` tags that are left over and shouldn't be subbed for the `*`.

Reply:
🤬 qualys...